docs-src/concepts/what-is-a-cloud-account-id.md

```diff
@@ -8,12 +8,56 @@ tags:
   - explanation
 ---
 
-A Temporal Cloud Account Id is a unique identifier for a customer for the entire time they use Temporal Cloud.
-Temporal Technologies assigns each Account Id, which is an opaque code of five or six alphanumeric characters, such as `f45a2`.
-The Account Id is shown in the upper-right corner of Workflows and Schedules listed in the Web UI.
+A Temporal Cloud Account Id is a unique customer identifier assigned by Temporal Technologies.
+Each Id is a short string of numbers and letters like `f45a2`, at least five characters long.
+This account identifier is retained throughout the time each customer uses Temporal Cloud.
 
-:::info
+At times you may need to know a customer Account Id.
+Accessing the account's Namespaces provides an easy way to capture this information.
+Each Temporal Namespace use an Account Id suffix.
+This is the alphanumeric character string found after the period in any Temporal Cloud Namespace name.
 
-For more information on obtaining a Temporal Cloud Account Id, see our [Cloud account commands](/cloud/tcld/account/get) page.
+You can retrieve an Account Id from the [Temporal Cloud](https://cloud.temporal.io) Web UI or by using the `tcld` utility at a command line interface (CLI).
+Follow these steps.
 
-:::
+**From the Temporal Cloud Web UI:**
+
+1. Log into Temporal Cloud.
+2. Navigate to the Namespace list page from the left-side vertical navigation.
+3. View the Namespaces listed for the user.
+   Each Namespace uses an Account Id suffix.
+
+**From the Command Line:**
+
+1. Use the `tcld` utility to log into an account.
+   A Web page automatically opens for authentication.
+
+       $ tcld login
+
+2. Ensure the user code shown by the CLI utility matches the code shown in the Web browser.
+   Then, click Confirm in the browser to continue.
+
+       $ tcld login
+       Login via this url: https://login.tmprl.cloud/activate?user_code=KTGC-ZPWQ
+
+   After confirmation, Web feedback lets you know that the CLI "device" is now connected.
+
+3. Return to the command line.
+   Issue the following command.
+
+       $ tcld namespace list
+
+   The CLI tool returns a short JSON packet with your namespace information.
+   This is the same list found in the Temporal Cloud Web UI Namespaces list.
+   Like the browser version, each Namespace uses an Account Id suffix.
+
+       $ tcld namespace list
+       {
+        "namespaces": [
+       		"your-namespace.123def",
+       		"another-namespace.123def"
+        ],
+       	"nextPageToken": ""
+       }
```

Comment:
These would be two different code blocks - 
1 for input and 1 for expected results

Reply:
Updated as suggested. New push coming.